brainfuck/executor_test.go

```diff
@@ -0,0 +1,56 @@
+package brainfuck
+
+import (
+	"bytes"
+	"io"
+	"os"
+	"testing"
+)
+
+func captureStdout(f func()) string {
+	old := os.Stdout
+	r, w, _ := os.Pipe()
+	os.Stdout = w
+
+	f()
+
+	w.Close()
```

Comment:
you can use `defer` here

Reply:
Now this can't be made. Because of handing error returned by Close().